src/test/scala/com/microsoft/azure/sqldb/spark/utils/SQLServerTestUtils.scala

```diff
@@ -0,0 +1,185 @@
+package com.microsoft.azure.sqldb.spark.utils
+
+import java.sql.{Connection, DriverManager}
+
+import com.microsoft.azure.sqldb.spark.config.{Config, SqlDBConfig}
+import com.microsoft.azure.sqldb.spark.connect.ConnectionUtils.{createConnectionProperties, createJDBCUrl}
+
+
+private[spark] class SQLServerTestUtils {
+  val DRIVER_NAME = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
+
+
+  private def getConnection(writeConfig: Config, connectToDB: Boolean): Connection = {
+    val url = writeConfig.get[String](SqlDBConfig.URL).get
+    val db = writeConfig.get[String](SqlDBConfig.DatabaseName).get
+    val properties = createConnectionProperties(writeConfig)
+    val user = writeConfig.get[String](SqlDBConfig.User).get
+    val password = writeConfig.get[String](SqlDBConfig.Password).get
+    val port = writeConfig.get[String](SqlDBConfig.PortNumber).getOrElse("1433")
+    if(db.equals(null) || url.equals(null)){
+      return null
+    }
+    Class.forName(DRIVER_NAME)
+    var dbConnect = ""
+    if(connectToDB){
+      dbConnect = ";database="+db
+    }
+    var conn: Connection = DriverManager.getConnection(createJDBCUrl(url, Some(port))+ dbConnect, user, password)
```

Comment:
Similar thing to other comment, try reusing existing connection code so that we can support all the config cases like AAD authentication etc.